resources/content/schema/base/snapshot.json

```diff
@@ -0,0 +1,14 @@
+{
+    "resourceFields": {
+        "volumeUUID": {
+            "type": "string",
+            "nullable": false
+        },
+        "snapshotUUID": {
```

Comment:
I would prefer snapshotUUID, since (well) golang doesn't like the word Id...

Others are addressed